components/uuf-renderablecreator-hbs/src/test/java/org/wso2/carbon/uuf/handlebars/HbsHelperTest.java

```diff
@@ -131,7 +133,8 @@ public void testTitle() {
     @Test
     public void testTitleWithMultipleParameters() {
         RequestLookup requestLookup = createRequestLookup();
-        createRenderable("{{title \"page\" \"-\" \"title\"}}").render(null, createLookup(), requestLookup, createAPI());
+        createRenderable("{{title \"page\" \"-\" \"title\"}}").render(null, createLookup(), requestLookup,
```

Comment:
Seems like this line doesn't exceed 120 limit. Hence, revert this formatting change.